publications/tasks.py

```diff
@@ -104,3 +110,38 @@ def harvest_oai_endpoint(url):
             parse_oai_xml_and_save_publications(response.content)
     except requests.exceptions.RequestException as e:
         print ("The requested URL is invalid or has bad connection.Please change the URL")
+
+def send_monthly_email(sent_by=None):
+    recipients = User.objects.values_list('email', flat=True)
+    last_month = now().replace(day=1) - timedelta(days=1)  # Get last month's last day
+    new_manuscripts = Publication.objects.filter(creationDate__month=last_month.month)
+
+    if not new_manuscripts.exists():
+        print("No new manuscripts found for this month. Skipping email.")
+        return
+
+    subject = "New Manuscripts This Month"
+    content = "Here are the new manuscripts:\n" + "\n".join([pub.title for pub in new_manuscripts])
```

Comment:
The message should also include links to the publications. Please create a new issue with this task, because currently we do not have a permalink for a publication in OPTIMAP.

Reply:
Created the issues for this: #102 #103